### ISSUE TITLE
Увеличение таймаутов

### ISSUE THREAD

**Issue by dartraiden** [gxcreator/i2pd-openwrt#3]
Полагаю, имеет смысл будущие версии собирать с [этим](https://github.com/PurpleI2P/i2pd/issues/345#issuecomment-177171901) патчем :)

**Comment by gxcreator** [comment]
Да, надо бы включить, только я там увеличил таймауты на случайные значения, неплохо было бы потестить и подобрать нужные для разного железа. Мне скоро должен приехать Xiaomi Mini, посмотрю как там будет работать.

**Comment by dartraiden** [comment]
На TL-WR1043NDv1 по дефолту всё печально (100% загрузка CPU). Да ещё и за провайдерским NAT.

Uptime: 01:25:31
Status: Firewalled
Tunnel creation success rate: 22%
Received: 3184K (439 Bps)
Sent: 6031K (1318 Bps)
Routers: 499 Floodfills: 371 LeaseSets: 0

**Comment by gxcreator** [comment]
На MR3040 такая же ситуация(там и проц такой же). Можно поробовать отключить проверку подписей, это немного ускорит работу(но хз как будет с т.з. секьюрности)

**Comment by dartraiden** [comment]
Проц этот, к слову, можно подразогнать, есть отзывы об успешном разгоне c 400 до 430 мегагерц.

**Comment by gxcreator** [comment]
Думаю это не сильно ему поможет
